src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
```

Comment:
Using 'this' causes the error to pop up. I guess that is because methods 'add', 'subtract' and so on called from 'operate' method like a callback functions.

Error log:
script.js:8 Uncaught TypeError: Cannot read properties of undefined (reading 'result')
    at add (script.js:8:7)
    at Object.operate (script.js:24:7)
    at script.js:44:4
add @ script.js:8
operate @ script.js:24
(anonymous) @ script.js:44

Reply:
cause you need to bind the callback in operate method